src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -0,0 +1,475 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import argparse
+import numpy as np
+from ecl.summary import EclSum
+
+
+DESCRIPTION = """
+Script to extract simulated welltest results from simulator output.
+Typically used to compare with output from a welltest analysis, eg using Cappa
```

Comment:
Cappa is spelled Kappa

Reply:
Ups, from this you may infer that the author is NOT a welltesting expert :)